jobwatch/skawatch.py

```diff
@@ -108,18 +110,20 @@ def main():
 
     jws = []
     jws.extend([
-        SkaJobWatch('aca_lts_eval', 2, errors=py_errs),
         SkaJobWatch('aca_hi_bgd_mon', 2, errors=py_errs,
                     filename='/proj/sot/ska/data/aca_hi_bgd_mon/logs/daily.0/aca_hi_bgd.log'),
         SkaJobWatch('acdc', 2, errors=py_errs),
-        SkaJobWatch('aimpoint_mon', 2, errors=py_errs),
+        SkaJobWatch('aimpoint_mon', 2, errors=py_errs,
+                    exclude_errors=['ErfaWarning']),
```

Comment:
Please do remember to revert this change in a few weeks when astropy 4.2 is included. Although it is benign from the processing perspective, it's good to see these in our processing as a precursor to users stumbling into the same problem.

Reply:
In this case, I was thinking that using 'aimpoint_mon' for sensitivity to ErfaWarning is not the right thing.  But if we don't have module testing in cxotime or some such that would clue us in, I can be more sensitive to it.  In this case, because aimpoint_mon is Python 2 code and we're not going to update astropy to 4.2 in Ska2 and I don't know when the aimpoint-mon code will be updated to Ska3,  it seems like the right fix is to shut up the warnings in Python 2 (by either getting a new file, or a live fix to ignore the warnings in aimpoint_mon or astropy in Ska2).

Reply:
With https://github.com/sot/aimpoint_mon/pull/21 merged it will not be *too* far in the future when we do the Py3 transition. Anyway, I don't mind shutting this up now, but I would rather not leave these ERFA suppression hooks in place indefinitely.

Reply:
Could I just plop this in the current Ska2 plot_aimpoint.py  or some such?
```
import warnings
from astropy._erfa.core import ErfaWarning
warnings.filterwarnings("ignore", category=ErfaWarning)
```

Reply:
Shutting up the not-flight-managed code directly seems like it would shut things up and have the side-effect that if the problem comes up again (with a ska3 install) we'd see it.

Reply:
I've cut the ErfaWarning here for now.